src/main/scala/net/ceedubs/ficus/readers/CaseInsensitiveEnumerationReader.scala

```diff
@@ -0,0 +1,7 @@
+package net.ceedubs.ficus.readers
+
+trait CaseInsensitiveEnumerationReader extends EnumerationReader {
```

Comment:
Why trait here? Why not an object so that users can use directly?

Reply:
Because `EnumerationReader` was trait already

Reply:
Thanks for merging this!